grammars/gfm.cson

```diff
@@ -774,4 +774,30 @@
     'end': '--\\s*>'
     'name': 'comment.block.gfm'
   }
+  {
+    'begin': '(^\\|)'
+    'end': '(\\|$)'
+    'captures':
+      '1':
+        'name': 'border.pipe.outer'
+      '2':
```

Comment:
It doesn't look like their is a second group in either the `begin` or `end` so I don't think this is needed here.